src/pybind/mgr/dashboard/controllers/pool.py

```diff
@@ -117,6 +117,8 @@ def set_key(key, value):
             CephService.send_command('mon', 'osd pool set', pool=pool, var=key, val=str(value))
 
         for key, value in kwargs.items():
+            if key == 'pool':
+               CephService.send_command('mon', 'osd pool rename', srcpool=pool, destpool=value)
```

Comment:
```
************* Module dashboard.controllers.pool
W:121, 0: Bad indentation. Found 15 spaces, expected 16 (bad-indentation)
```

@familyuu might want to address this issue.

Reply:
Yeah, I really want to solve this issue, but something is not completed, I have always working on it for to solve it. this PR may be submitted too anxious.

Reply:
> this PR may be submitted too anxious

No problem, just keep on evolving it!